homeassistant/components/surepetcare/sensor.py

```diff
@@ -19,10 +21,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def async_setup_entry(hass, entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
     """Set up Sure PetCare Flaps sensors."""
 
-    entities: list[SurepyEntity] = []
+    entities: list[SureBattery] = []
```

Comment:
Is there only a battery type sensor ? looks like quite a limitation.

Reply:
Yes, currently only battery sensor is supported